ios/RNPurchases.m

```diff
@@ -264,5 +266,13 @@ - (void)rejectPromiseWithBlock:(RCTPromiseRejectBlock)reject error:(NSError *)er
     };
 }
 
+- (NSString *)platformFlavor {
+    return @"react-native";
+}
+
+- (NSString *)platformFlavorVersion {
+    return @"3.3.0";
+}
```

Comment:
It might make sense somewhere in the future to create a delegate so that we can standardize the names of the methods in these classes, but I don't see it as important for now.